docs/upgrade.md

```diff
@@ -87,6 +87,14 @@ process, for example:
     wget https://packages.matrix.org/debian/pool/main/m/matrix-synapse-py3/matrix-synapse-py3_1.3.0+stretch1_amd64.deb
     dpkg -i matrix-synapse-py3_1.3.0+stretch1_amd64.deb
     ```
+# Upgrading to v1.88.0
+
+## Removal of `worker_replication_*` settings
+As mentioned in v1.84.0, these settings are deprecated and are being removed. Ensure you
```

Comment:
a link to the section showing you how to migrate would be useful

Reply:
Attempted fix in [c352219](https://github.com/matrix-org/synapse/pull/15860/commits/c352219d226ad6f95b6d3c50930543068cff9dce)